src/constants/pages.js

```diff
@@ -28,27 +28,67 @@ function generatePath(tree, parentPath = '') {
 
 export const docPages = generatePath([
   {
-    name: 'Introduction',
+    name: 'Welcome!',
     content: getDocUrl('demos/introduction.md')
   },
   {
-    name: 'Starting With a Map',
-    content: getDocUrl('demos/starting-with-map.md')
+    name: 'React Map GL',
+    children: [
+      {
+        name: 'Starting With a Map',
+        content: getDocUrl('demos/starting-with-map.md')
+      }
+    ]
   },
   {
-    name: 'Scatterplot Overlay',
-    content: getDocUrl('demos/scatterplot-overlay.md')
+    name: 'Deck GL',
+    children: [
+      {
+        name: 'Scatterplot Overlay',
+        content: getDocUrl('demos/scatterplot-overlay.md')
+      }
+    ]
   },
   {
-    name: 'Add Charts',
-    content: getDocUrl('demos/add-charts.md')
+    name: 'React Vis',
+    children: [
+      {
+        name: 'Add Charts',
+        content: getDocUrl('demos/add-charts.md')
+      },
+      {
+        name: 'Line Charts',
+        content: getDocUrl('demos/line-charts.md')
+      },
+      {
+        name: 'Scatterplot Charts',
+        content: getDocUrl('demos/scatterplot-charts.md')
+      }
+    ]
   },
   {
-    name: 'Line Charts',
-    content: getDocUrl('demos/line-charts.md')
-  },
-  {
-    name: 'Scatterplot Charts',
-    content: getDocUrl('demos/scatterplot-charts.md')
+    name: `Visualization guidelines`,
+    children: [
+      {
+        name: `Do - clear simple charts`,
+        path: 'simple',
```

Comment:
the router tries to create a route/path based on the name but chokes on things like quotes or dashes. what was happening was that several pages ended up having the same route. 
so I'm helping it by specifying a path explicitly.